### PR TITLE
feat: implement remember-me

### PR DESCRIPTION
**Pull request by Helias**
closes https://github.com/azerothcore/Keira3/issues/1215

### CODE REVIEW

File: src/app/main/connection-window/connection-window.component.spec.ts [hunk 1]
```diff
@@ -281,6 +292,20 @@ describe('ConnectionWindowComponent', () => {
...
 
+  describe('remember me checkbox', () => {
+    it('when isRemember return true it should call onConnect', waitForAsync(async () => {
+      const { page, component, connectionWindowService } = setup(false);
+      spyOn(component, 'onConnect');
+      connectionWindowService.getConfigs.and.returnValue(mockConfigsWithPass);
+      connectionWindowService.isRemember.and.returnValue(true);
+
+      page.detectChanges();
+      await page.whenStable();
```
Comment (FrancescoBorzi) [inline src/app/main/connection-window/connection-window.component.spec.ts:303]: where is the action?

File: src/app/main/connection-window/connection-window.service.ts [hunk 2]
```diff
@@ -45,6 +45,14 @@ export class ConnectionWindowService {
     return configs;
   }
 
+  saveRememberPreference(rememberMe: boolean): void {
+    this.localStorageService.setItem('rememberMe', String(rememberMe));
+  }
+
+  isRemember(): boolean {
```
Comment (FrancescoBorzi) [inline src/app/main/connection-window/connection-window.service.ts:52]: `isRemember` does not make sense, maybe a better name can be `isRememberMeEnabled`

**Review by FrancescoBorzi — APPROVED** [review]

File: src/app/main/connection-window/connection-window.component.spec.ts (hunk 1 above)
Comment (Helias) [reply src/app/main/connection-window/connection-window.component.spec.ts:303]: in the OnInit IIRC

File: src/app/main/connection-window/connection-window.service.ts (hunk 2 above)
Comment (Helias) [reply src/app/main/connection-window/connection-window.service.ts]: done

**Pull request merged**